app/services/catkey.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class Catkey
+  SYMPHONY = 'symphony'
+  PREVIOUS_CATKEY = 'previous_catkey'
+
+  def self.deserialize(model)
+    new(model).deserialize
+  end
+
+  def self.serialize(model, catkey)
+    new(model).serialize(catkey)
+  end
+
+  def initialize(model)
+    @model = model
+  end
+
+  def deserialize
+    symphony_links.join(', ')
+  end
+
+  # If there was already a catkey in the record, store that in the "previous" spot (assuming there is no change)
+  # @return [Array<Hash>] a list of catalog links
```

Comment:
Can you document the supported format in Rdoc?

Reply:
@justinlittman the format of the catkey parameter?